### PR TITLE
T4 actuators and sensors - serial servos with position feedback

### PR DESCRIPTION
**Pull request by dewagter**
Replaces #3295

### CODE REVIEW

File: conf/modules/actuators_t4.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  <dep>
+    <depends>actuators</depends>
+    <provides>actuators</provides>
+  </dep>
+  <header>
+    <file name="actuators_t4.h"/>
+  </header>
+  <makefile>
+    <file_arch name="actuators_t4_arch.c"/>
+<!--    <file_arch name="actuators_shared_arch.c" cond="ifeq ($(ARCH), stm32)"/>-->
```
Comment (EwoudSmeur) [inline conf/modules/actuators_t4.xml:18]: should this be included or deleted?

File: conf/airframes/examples/example_t4_actuators_fw.xml [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+
+    <!--    <servo name="S_ID5"   no="4" min="20" neutral="510" max="1000"/>-->
+<!--    <servo name="S_ID6"   no="5" min="1000" neutral="510" max="20"/>-->
+<!--    <servo name="S_ID7"   no="6" min="20" neutral="510" max="1000"/>-->
+<!--    <servo name="S_ID8"   no="7" min="20" neutral="510" max="1000"/>-->
+<!--    <servo name="S_ID9"   no="8" min="20" neutral="510" max="1000"/>-->
+<!--    <servo name="S_ID10"   no="9" min="20" neutral="510" max="1000"/>-->
+  </servos>
+
+ <!--<servos driver="t4">--><!-- At the moment only one T4 board is supported, but feel free to enhance -->
```
Comment (EwoudSmeur) [inline conf/airframes/examples/example_t4_actuators_fw.xml:174]: If only one T4 board is supported, I think this commented section can be removed

File: conf/airframes/examples/example_t4_actuators_fw.xml
Comment (EwoudSmeur) [inline conf/airframes/examples/example_t4_actuators_fw.xml:1]: There are many commented parts in this airframe file that can be cleaned up

**Review by EwoudSmeur — COMMENTED** [review]
Great development! 

I'm not sure what the effects of the board files and image_maxsize etc is, maybe @fvantienen can comment on this?

**Comment by OpenUAS** [comment]
A full refactored and integrated with current master is worked at the very moment... this branch will be removed.

**Pull request closed without merging**